chef-utils/lib/chef-utils/dsl/train_helpers.rb

```diff
@@ -57,6 +57,14 @@ def file_open(*args, &block)
         end
       end
 
+      def home
+        if __transport_connection
+          __transport_connection.run_command("echo $HOME").stdout&.strip
+        else
+          ENV["$HOME"]
+        end
+      end
```

Comment:
This shouldn't be added here, we pull in a train-aware `shell_out` into Chef::Mixin::ShellOut so that should be used directly with `shell_out("echo $HOME").stdout&.strip` in the caller.

Reply:
sounds reasonable to me will look into it thanks!